RetailSuiteAccounting/Operation/11_CreateBalanceCarriedForward.md

```diff
@@ -0,0 +1,51 @@
+[!!User Interface balance carried forward](../UserInterface/01_Book.md#balance-carried-forward)  
+[!!User Interface settings balance carried forward](../UserInterface/02h_BalanceCarriedForward.md)  
+
+
+# Create a balance carried forward
+
+At the turn of the fiscal year, any outstanding balances must be carried forward to the next one. Closing balances will then be transferred as opening balances into the new fiscal year.
+
+The *Balance carried forward* menu entry of the *EDIT* context menu in the *BOOKINGS* allows to transfer outstanding account balances and open items from one fiscal year to the next one. This is possible for both personal (debtors and creditors) accounts and impersonal accounts, for example bank or tax accounts.
+
+Special journals are preconfigured in the system to carry balances and open items forward. For example, journal 98 contains all balances carried forward for debtors open items. These settings can be completely customized. For detailed information, see [User Interface settings balance carried forward](../UserInterface/02h_BalanceCarriedForward.md).
+
+#### Prerequisites
+
+- The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).
+- A new valid fiscal years is created, see [Create a fiscal year](../Integration04_ManageFiscalYear.md#create-a-fiscal-year).
+- A fiscal year is selected, see [Select fiscal year](./01_SelectFiscalYear.md).
+- The balance carried forward function is configured, see [User Interface settings balance carried forward](../UserInterface/02h_BalanceCarriedForward.md).
```

Comment:
Ich schätze, hier brauchen wir noch nen Kapitel in Integration, wie man das konfiguriert. Im User Interface wird das nämlich nicht wirklich deutlich...

Reply:
Hmm, vielleicht, aber der Balance Carried Forward Konten sind eigentlich vorkonfiguriert. Man kann sie ändern/anders einstellen, aber nicht zwingend notwendig, sofern ich weiß. Ich packe es in die offenen Fragen.